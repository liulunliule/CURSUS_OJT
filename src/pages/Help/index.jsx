import React, { useState } from "react";
import "./index.scss";
import Search from "../../assets/img/searchIcon.png";
import Wallet from "../../assets/img/wallet.png";
import Start from "../../assets/img/start.png";
import { useSelector } from "react-redux";

const Help = () => {
  const [activeTab, setActiveTab] = useState("nav-instructor");
  const handleTabClick = (tabId) => {
    setActiveTab(tabId); // Cập nhật state activeTab với tabId được click
  };
  const isShowAll = useSelector((state) => state.savedCourse.isShowAll);
  return (
    <div className={`wrapper12 _help ${isShowAll ? "active" : ""}`}>
      <div className="help-background">
        <div className="container-fluid">
          <div className="row">
            <div className="col-lg-12">
              <div className="helpSection">
                <div className="row justify-content-md-center">
                  <div className="col-md-6">
                    <div className="help_title">
                      <h2>How may we help you?</h2>
                      <div>
                        <div className="search-focus">
                          <div className="search-input">
                            <img src={Search} className="search_icon" />
                            <input
                              className="explore-search"
                              type="text"
                              placeholder="Search for solutions"
                            ></input>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>

      <div className="instu-tab">
        <div className="container">
          <div className="row">
            <div className="col-lg-12">
              <div className="instu-section">
                <div className="ins-or-stu">
                  <nav>
                    <div
                      className="nav nav-tabs tab_crse"
                      id="nav-tab"
                      role="tablist"
                    >
                      <a
                        className={`nav-item nav-link ${
                          activeTab === "nav-instructor" ? "active" : ""
                        }`}
                        id="nav-instructor-tab"
                        data-bs-toggle="tab"
                        href="#nav-instructor"
                        role="tab"
                        aria-selected={activeTab === "nav-instructor"}
                        onClick={() => handleTabClick("nav-instructor")}
                      >
                        Instructor
                      </a>
                      <a
                        className={`nav-item nav-link ${
                          activeTab === "nav-student" ? "active" : ""
                        }`}
                        id="nav-student-tab"
                        data-bs-toggle="tab"
                        href="#nav-student"
                        role="tab"
                        aria-selected={activeTab === "nav-student"}
                        onClick={() => handleTabClick("nav-student")}
                      >
                        Student
                      </a>
                    </div>
                  </nav>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>

      <div className="instu-content">
        <div className="container-fluid">
          <div className="row">
            <div className="col-lg-12">
              <div className="course_tab_content">
                <div className="tab-content" id="nav-tabContent">
                  <div
                    className={`tab-pane fade ${
                      activeTab === "nav-instructor" ? "active show" : ""
                    }`}
                    id="nav-instructor"
                    role="tabpanel"
                    aria-labelledby="nav-instructor-tab"
                  >
                    <div className="instr-cont">
                      <div className="ins-topic">
                        <h3>Select a topic to search for help</h3>
                      </div>
                      <div className="helpCont">
                        <div className="row">
                          {content.map((d) => (
                            <div className="col-md-4">
                              <a href="#" className="help-props">
                                <div className="value_props100">
                                  <div className="value_icon">
                                    <img src={d.icon} />
                                  </div>
                                  <div className="value_contents">
                                    <h4>{d.slogan}</h4>
                                    <p>{d.desc}</p>
                                  </div>
                                </div>
                              </a>
                            </div>
                          ))}
                        </div>
                      </div>
                    </div>

                    <div className="faqs">
                      <div className="ins-topic">
                        <h3>Frequently Asked Questions</h3>
                      </div>
                      <div className="faq-section">
                        <div className="row">
                          {questions.map((d) => (
                            <div className="col-md-4">
                              <a href="#" className="value_props1000">
                                {d.faq}
                              </a>
                            </div>
                          ))}
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className={`tab-pane fade ${
                      activeTab === "nav-student" ? "active show" : ""
                    }`}
                    id="nav-student"
                    role="tabpanel"
                    aria-labelledby="nav-student-tab"
                  >
                    <div className="instr-cont">
                      <div className="ins-topic">
                        <h3>Select a topic to search for help</h3>
                      </div>
                      <div className="helpCont">
                        <div className="row">
                          {content1.map((d) => (
                            <div className="col-md-4">
                              <a href="#" className="help-props">
                                <div className="value_props100">
                                  <div className="value_icon">
                                    <img src={d.icon} />
                                  </div>
                                  <div className="value_contents">
                                    <h4>{d.slogan}</h4>
                                    <p>{d.desc}</p>
                                  </div>
                                </div>
                              </a>
                            </div>
                          ))}
                        </div>
                      </div>
                    </div>

                    <div className="faqs">
                      <div className="ins-topic">
                        <h3>Frequently Asked Questions</h3>
                      </div>
                      <div className="faq-section">
                        <div className="row">
                          {questions1.map((d) => (
                            <div className="col-md-4">
                              <a href="#" className="value_props1000">
                                {d.faq}
                              </a>
                            </div>
                          ))}
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

const content = [
  {
    slogan: "Payments",
    desc: "Understand the revenue share and how to receive payments",
    icon: Wallet,
  },
  {
    slogan: "Selling & Promotion",
    desc: "Learn about the announcement and promotional tools",
    icon: Wallet,
  },
  {
    slogan: "Quality Standards",
    desc: "Learn what it takes to create a high quality course",
    icon: Wallet,
  },
  {
    slogan: "Course Building",
    desc: "Build your course curriculum and landing page",
    icon: Wallet,
  },
  {
    slogan: "Course Management",
    desc: "Maintain your course and engage with students",
    icon: Wallet,
  },
  {
    slogan: "Trust & Safety",
    desc: "Policy and copyright questions and guidance",
    icon: Wallet,
  },
];

const questions = [
  {
    faq: "Promote Your Course With Coupons and Referral Links",
  },
  {
    faq: "Cursus Course Quality Checklist",
  },
  {
    faq: "Instructor Revenue Share",
  },
  {
    faq: "Instructor Promotional Agreements and Cursus Deals",
  },
  {
    faq: "How to Become an Instructor: FAQ",
  },
  {
    faq: "How to Select Your Payout Method and Become a Premium Instructor",
  },
];

const content1 = [
  {
    slogan: "Getting Started",
    desc: "Learn how Cursus works and how to start learning",
    icon: Start,
  },
  {
    slogan: "Account/Profile",
    desc: "Manage your account settings",
    icon: Start,
  },
  {
    slogan: "Troubleshooting",
    desc: "Experiencing a bug? Check here",
    icon: Start,
  },
  {
    slogan: "Course Taking",
    desc: "Everything about taking a course on Udemy",
    icon: Start,
  },
  {
    slogan: "Purchase/Refunds",
    desc: "Learn about coupons, how to send gifts, and refunds",
    icon: Start,
  },
  {
    slogan: "Mobile",
    desc: "On the go? Learn about our mobile app",
    icon: Start,
  },
];

const questions1 = [
  {
    faq: "Lifetime Access",
  },
  {
    faq: "Cursus FAQ",
  },
  {
    faq: "Downloading Courses",
  },
  {
    faq: "Certificate of Completion",
  },
  {
    faq: "Refund a Course",
  },
  {
    faq: "How to Solve Payment Issues",
  },
];
export default Help;
