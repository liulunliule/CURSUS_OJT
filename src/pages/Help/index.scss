._help {
    // padding: 60px 0 0 !important;
    margin-top: 60px;
}

.help-background {
    float: left;
    width: 100%;
    padding: 80px 0;
    background: #333;
    position: relative;
}

.helpSection {
        float: left;
        width: 100%;
}

.help_title h2{
        font-size: 30px;
        font-weight: 600;
        font-family: 'Roboto', sans-serif;
        color: #fff;
        text-align: center;
        margin-bottom: 30px;
}

.search-focus {
    position: relative;
    font-size: 1em;
}

.search-input {
        position: relative;
        font-weight: 400;
        font-style: normal;
        display: inline-flex;
        color: rgba(0,0,0,.87);
        font-size: 14px !important;
        width: 100%;
}

.explore-search {
        padding-left: 1.2em!important;
        padding-right: 1em!important;
        height: 50px;
        font-size: 14px;
        font-weight: 400;
        font-family: 'Roboto', sans-serif;
        border-radius: 3px !important;
        flex: 1 0 auto;
        background: #fff !important;
        border: 1px solid #e5e5e5 !important;
        color: #333 !important;
        box-shadow: inset 0px 1px 4px rgba(0, 0, 0, 0.02) !important;
}

.search_icon {
    right: auto;
    margin-left: 10px;
    margin-top: 5px;
    width: 40px;
    height: 40px;

}

.instu-tab {
    float: left;
    width: 100%;
    padding: 20px 0 0;
    background: #fff;
    // margin-bottom: 17px;
    margin-top: 17px;
}

.instu-section {
    float: left;
    width: 100%;
    margin-top: 10px;
}

.ins-or-stu {
    justify-content: center;
    display: flex;
}

.instu-content {
        float: left;
        width: 100%;
        background-color: #F8F8F8;
}

.instr-cont {
        float: left;
        width: 100%;
        margin-top: 0px;
}

.ins-topic {
        float: left;
        width: 100%;
        margin-top: 30px;
        margin-bottom: 20px;
        h3 {
            font-size: 20px;
            font-weight: 500;
            font-family: 'Roboto', sans-serif;
            text-align: left;
            margin-bottom: 15px;
            color: #333;
            float: left;
            margin-left: 20px;
        }
}

.helpCont {
    float: left;
    width: 100%;
    margin-top: 20px;
}

.help-props {
    background: #fff;
    border-radius: 3px;
    padding: 20px;
    border: 1px solid #efefef;
    box-shadow: none;
    flex: 1;
    float: left;
    width: 92%;
    margin-bottom: 30px;
    text-align: center;
    height: 225px;
    margin-left: 15px;
    margin-right: 20px;
}

.help_tabs {
    float: left;
    width: 100%;
    margin-top: 10px;
}

.value_props100 {
    background: #fff;
    // border-radius: 3px;
    padding: 20px;
    // border: 1px solid #efefef;
    box-shadow: none;
    flex: 1;
    float: left;
    width: 100%;
    margin-bottom: 30px;
    text-align: center;
    height: 200px;
}

.value_contents {
    float: left;
    width: 100%;
    margin-top: 20px;
}

.value_contents h4 {
    font-size: 18px;
    margin-bottom: 9px;
    display: block;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    color: #333;
    text-align: center;
}

.value_contents p {
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    font-size: 14px;
    text-align: left;
    line-height: 26px;
    color: #686f7a;
    text-align: center;
}

.faqs {
    float: left;
    width: 100%;
}

.faq-section {
    float: left;
    width: 100%;
    margin-top: 20px;
}

.value_props1000 {
    background: #fff;
    border-radius: 3px;
    border: 1px solid #efefef;
    padding: 20px;
    box-shadow: none;
    flex: 1;
    float: left;
    color: #333 !important;
    width: 92%;
    font-size: 14px;
    line-height: 26px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 30px;
    text-align: center;
    margin-left: 15px;
    margin-right: 20px;
}