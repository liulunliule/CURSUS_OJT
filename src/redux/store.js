import { combineReducers, configureStore } from "@reduxjs/toolkit";
import savedCourseReducer from "./features/savedCourseSlice";
import blogReducer from './features/ourBlogSlice';
import officesReducer from './features/officesSlice';
import pressReducer from './features/pressSlice';
import payoutReducer from './features/payoutSlice';
import creditsReducer from "./features/creditsSlice";
import certificateReducer from "./features/myCertificateSlice";
import myProfileReducer from "./features/myProfileSlice";
import myHeaderReducer from "./features/myHeaderSlice";
import mySearchReducer from "./features/mySearchSlice";
import myStudioReducer from "./features/myStudioSlice";
import allInstructorReducer from "./features/allInstructorSlice";
import reviewFeedbackReducer from "./features/reviewfeedbackSlice";
import feedbackReportReducer from "./features/feedbackvsReportSlice";
import notificationsSettingReducer from "./features/NotificationsSettingSlice";
import privacySettingReducer from "./features/PrivacySettingSlice";
import userSlice from "./features/userSlice";
import certificationTestPageReducer from "./features/certificationTestPageSlice";
import certificationTestResultReducer from "./features/certificationTestResultSlice";
import analyticsReducer from "./features/analyticsSlice";
import storage from "redux-persist/lib/storage";
import { persistReducer, persistStore } from "redux-persist";
import courseSlice from "./features/courseSlice";
import checkoutReducer from './features/checkoutSlice';
import earningReducer from './features/earningSlice';
import statementReducer from './features/statementSlice';
import shoppingCartReducer from './features/shoppingCartSlice';

const persistConfig = {
    key: "root",
    storage,
    whitelist: [
        "user",
        "savedCourse",
        "certificate",
        "course",
        "myProfile",
        "myHeader",
        "mySearch",
        "myStudio",
        "allInstructor",
        "reviewFeedback",
        "feedbackReport",
        "notificationsSetting",
        "privacySetting",
    ],
};

const rootReducer = combineReducers({
    user: userSlice,
    savedCourse: savedCourseReducer,
    certificate: certificateReducer,
    certificationTestPage: certificationTestPageReducer,
    certificationTestResult: certificationTestResultReducer,
    analytics: analyticsReducer,
    course: courseSlice,
    myProfile: myProfileReducer,
    myHeader: myHeaderReducer,
    mySearch: mySearchReducer,
    myStudio: myStudioReducer,
    allInstructor: allInstructorReducer,
    reviewFeedback: reviewFeedbackReducer,
    feedbackReport: feedbackReportReducer,
    checkout: checkoutReducer,
    earnings: earningReducer,
    statement: statementReducer,
    credits: creditsReducer,
    payout: payoutReducer,
    blogs: blogReducer,
    offices: officesReducer,
    press: pressReducer,
    notificationsSetting: notificationsSettingReducer,
    privacySetting: privacySettingReducer,
    shoppingCart: shoppingCartReducer,
});

const persistedReducer = persistReducer(persistConfig, rootReducer);

export const store = configureStore({
    reducer: persistedReducer,
});

export const persistor = persistStore(store);
