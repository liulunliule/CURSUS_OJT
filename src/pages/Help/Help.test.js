import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router-dom'; // Import MemoryRouter
import { store } from '../../redux/store';
import Help from '.';

test('test verification page', () => {
  const tree = renderer.create(
    <Provider store={store}>
        <MemoryRouter>
            <Help />
        </MemoryRouter>
    </Provider>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});