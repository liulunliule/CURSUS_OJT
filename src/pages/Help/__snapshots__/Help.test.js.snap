// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test verification page 1`] = `
<div
  className="wrapper12 _help "
>
  <div
    className="help-background"
  >
    <div
      className="container-fluid"
    >
      <div
        className="row"
      >
        <div
          className="col-lg-12"
        >
          <div
            className="helpSection"
          >
            <div
              className="row justify-content-md-center"
            >
              <div
                className="col-md-6"
              >
                <div
                  className="help_title"
                >
                  <h2>
                    How may we help you?
                  </h2>
                  <div>
                    <div
                      className="search-focus"
                    >
                      <div
                        className="search-input"
                      >
                        <img
                          className="search_icon"
                          src="searchIcon.png"
                        />
                        <input
                          className="explore-search"
                          placeholder="Search for solutions"
                          type="text"
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="instu-tab"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-lg-12"
        >
          <div
            className="instu-section"
          >
            <div
              className="ins-or-stu"
            >
              <nav>
                <div
                  className="nav nav-tabs tab_crse"
                  id="nav-tab"
                  role="tablist"
                >
                  <a
                    aria-selected={true}
                    className="nav-item nav-link active"
                    data-bs-toggle="tab"
                    href="#nav-instructor"
                    id="nav-instructor-tab"
                    onClick={[Function]}
                    role="tab"
                  >
                    Instructor
                  </a>
                  <a
                    aria-selected={false}
                    className="nav-item nav-link "
                    data-bs-toggle="tab"
                    href="#nav-student"
                    id="nav-student-tab"
                    onClick={[Function]}
                    role="tab"
                  >
                    Student
                  </a>
                </div>
              </nav>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="instu-content"
  >
    <div
      className="container-fluid"
    >
      <div
        className="row"
      >
        <div
          className="col-lg-12"
        >
          <div
            className="course_tab_content"
          >
            <div
              className="tab-content"
              id="nav-tabContent"
            >
              <div
                aria-labelledby="nav-instructor-tab"
                className="tab-pane fade active show"
                id="nav-instructor"
                role="tabpanel"
              >
                <div
                  className="instr-cont"
                >
                  <div
                    className="ins-topic"
                  >
                    <h3>
                      Select a topic to search for help
                    </h3>
                  </div>
                  <div
                    className="helpCont"
                  >
                    <div
                      className="row"
                    >
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="wallet.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Payments
                              </h4>
                              <p>
                                Understand the revenue share and how to receive payments
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="wallet.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Selling & Promotion
                              </h4>
                              <p>
                                Learn about the announcement and promotional tools
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="wallet.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Quality Standards
                              </h4>
                              <p>
                                Learn what it takes to create a high quality course
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="wallet.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Course Building
                              </h4>
                              <p>
                                Build your course curriculum and landing page
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="wallet.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Course Management
                              </h4>
                              <p>
                                Maintain your course and engage with students
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="wallet.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Trust & Safety
                              </h4>
                              <p>
                                Policy and copyright questions and guidance
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  className="faqs"
                >
                  <div
                    className="ins-topic"
                  >
                    <h3>
                      Frequently Asked Questions
                    </h3>
                  </div>
                  <div
                    className="faq-section"
                  >
                    <div
                      className="row"
                    >
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          Promote Your Course With Coupons and Referral Links
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          Cursus Course Quality Checklist
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          Instructor Revenue Share
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          Instructor Promotional Agreements and Cursus Deals
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          How to Become an Instructor: FAQ
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          How to Select Your Payout Method and Become a Premium Instructor
                        </a>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-labelledby="nav-student-tab"
                className="tab-pane fade "
                id="nav-student"
                role="tabpanel"
              >
                <div
                  className="instr-cont"
                >
                  <div
                    className="ins-topic"
                  >
                    <h3>
                      Select a topic to search for help
                    </h3>
                  </div>
                  <div
                    className="helpCont"
                  >
                    <div
                      className="row"
                    >
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="start.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Getting Started
                              </h4>
                              <p>
                                Learn how Cursus works and how to start learning
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="start.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Account/Profile
                              </h4>
                              <p>
                                Manage your account settings
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="start.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Troubleshooting
                              </h4>
                              <p>
                                Experiencing a bug? Check here
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="start.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Course Taking
                              </h4>
                              <p>
                                Everything about taking a course on Udemy
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="start.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Purchase/Refunds
                              </h4>
                              <p>
                                Learn about coupons, how to send gifts, and refunds
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="help-props"
                          href="#"
                        >
                          <div
                            className="value_props100"
                          >
                            <div
                              className="value_icon"
                            >
                              <img
                                src="start.png"
                              />
                            </div>
                            <div
                              className="value_contents"
                            >
                              <h4>
                                Mobile
                              </h4>
                              <p>
                                On the go? Learn about our mobile app
                              </p>
                            </div>
                          </div>
                        </a>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  className="faqs"
                >
                  <div
                    className="ins-topic"
                  >
                    <h3>
                      Frequently Asked Questions
                    </h3>
                  </div>
                  <div
                    className="faq-section"
                  >
                    <div
                      className="row"
                    >
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          Lifetime Access
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          Cursus FAQ
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          Downloading Courses
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          Certificate of Completion
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          Refund a Course
                        </a>
                      </div>
                      <div
                        className="col-md-4"
                      >
                        <a
                          className="value_props1000"
                          href="#"
                        >
                          How to Solve Payment Issues
                        </a>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
